src/smoosh/test/smoosh_tests.erl

```diff
@@ -239,24 +239,33 @@ t_checkpointing_works(DbName) ->
     setup_db_compactor_intercept(),
     {ok, _} = delete_doc(DbName, <<"doc1">>),
     ok = wait_to_enqueue(DbName),
+    ?debugHere,
     CompPid = wait_db_compactor_pid(),
     ChanPid = get_channel_pid("ratio_dbs"),
     config:set("smoosh", "persist", "true", false),
     meck:reset(smoosh_channel),
     ChanPid ! checkpoint,
     % Wait for checkpoint process to exit
+    ?debugHere,
     ok = wait_normal_down(),
+    ?debugHere,
+    ?debugHere,
```

Comment:
what is the goal of having two consecutive debug statements?

Reply:
Doh, that's a silly copy-paste error.